crowdsec-docs/docs/appsec/hooks.md

```diff
@@ -16,30 +16,46 @@ The three phases are:
 
 Hooks are configured in your `appsec-config` file.
 
-Except for the `on_load` hook, all hooks support a `filter` parameter and an `apply` parameter (`on_load` only has `apply`).
+`on_load` hook only supports `apply`, while other hooks support `filter` and `apply` parameters.
 
 Both `filter` and `apply` of the same phase have access to the same helpers.
 
 Except for `on_load`, hooks can be called twice per request: once for in-band processing and once for out-of-band processing, thus it is recommended to use the `IsInBand` and `IsOutBand` variables to filter the hook.
 
+
+Hooks have the following format:
+
+```yaml
+on_match:
+  - filter: 1 == 1
```

Comment:
Maybe we should show an example of using `IsInBand && 1==1` just so they know how to properly filter on bands?